docs: Update information on extending Superset assets (#226)
openedx/openedx-aspects: docs/how-tos/superset_extra_assets.rst

Superset extra assets
*********************

> Developers user the `superset-extra-assets` patch to add extra assets to Superset and those

Reviewer: ```suggestion
Developers can use the `superset-extra-assets` patch to add extra assets to Superset and those
```

with the attributes that Superset expects for each asset type. See 
`assets <https://github.com/openedx/tutor-contrib-aspects/tree/main/tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets>`_ 
for examples of asset yaml declarations.

In most cases, however, you will want to develop Superset dashboards, charts, datasets, and 
sometimes even databases in the Superset UI and import them into a plugin that can be managed
in source control. As of Aspects 1.0.3 use the same command to import assets into a Tutor 
plugin that we use to maintain Aspects itself.

You can use the `Tutor Plugin Cookiecutter <https://github.com/overhangio/cookiecutter-tutor-plugin/>`_ 
to create a skeleton plugin. Then you can create a patch file that will include all of your 
Superset assets into Aspects when the plugin is enabled. 

The file name is the patch name ``<your_extension>/patches/superset-extra-assets``:

.. code-block:: 

    {% for file in "<your_extension>/superset-assets/"|walk_templates %}
    ---
    _file_name: {{ file.split('/')[-1] }}
    {% include file %}
    {% endfor %}

This recursively includes each file in the superset-assets directory of your extension's 

> tempates directory, when ``tutor config save is run``. For example 

Reviewer 2: ```suggestion
tempates directory, when ``tutor config save`` is run. For example 
```

``<your_extension>/templates/superset-assets/*``.

> Now you can export your Superset dashboard, which it will download to your local computer

Reviewer 2: ```suggestion
Now you can export your Superset dashboard, which will download to your local computer
```
